README.md

```diff
@@ -128,6 +128,18 @@ container and output it to local directory.
 
 With `TARGET_GOOS` environment variable, you can also cross compile the binary.
 
+You can also build your OS by docker with the command below:
```

Comment:
The instructions above already say how you can use Docker to build a binary from source, though it could be enhanced with an example.

For a Mac:
```
TARGET_GOOS=darwin make docker
```

For Windows:
```
TARGET_GOOS=windows make docker
```

I'd be happy to merge a pull request that extended the existing instructions.